src/main/java/sirius/web/health/SystemController.java

```diff
@@ -168,6 +159,16 @@ public void metric(WebContext ctx, String key) {
      * @param ctx the request being handled
      */
     @Routed("/system/metrics")
+    @PublicService(apiName = "health", label = "Metrics API", description = """
+            Provides all collected metrics in Prometheus compatible format.
+            """, format = Format.RAW, exampleResponse = """
+            # HELP sirius_node_state Node State
+            # TYPE sirius_node_state gauge
+            sirius_node_state 0.0
+            # HELP sirius_http_open_connections HTTP Open Connections
+            # TYPE sirius_http_open_connections gauge
+            sirius_http_open_connections 7.0
+            """)
```

Comment:
Question: These will not be translated?

Reply:
Nope - this is actually an old service and just a new piece of documentation. These are auto-generated for prometheus using the available metadata.